dspace-api/src/main/java/org/dspace/versioning/VersioningServiceImpl.java

```diff
@@ -165,21 +191,32 @@ public Version createNewVersion(Context context, VersionHistory history, Item it
             throw new RuntimeException(e.getMessage(), e);
         }
     }
+    
+    @Override
+    public List<Version> getVersionsByHistory(Context c, VersionHistory vh) throws SQLException {
```

Comment:
I would add a comment on this method to state that only versions with an item attached to it are returned and that is should only be used in that manner.